src/math/SpatialCell.cc

```diff
@@ -86,9 +86,8 @@ SpatialCell::SpatialCell(std::string const& label, ///< string representing "nam
     _candidateList(candidateList),
     _ignoreBad(true)
 {
-    lsst::pex::logging::TTrace<3>("lsst.afw.math.SpatialCell",
-                                  "Cell %s : created with %d candidates",
-                                  this->_label.c_str(), this->_candidateList.size());
+    LOGL_DEBUG("TRACE2.afw.math.SpatialCell", "Cell %s : created with %d candidates",
+               this->_label.c_str(), this->_candidateList.size());
```

Comment:
Agree with what you said. Under `afw.math` loggers, these use one tracing/debugging level: 
`afw.math.KernelFormatter`
`afw.math.LeastSquares`
`afw.math.SpatialCell`
So no prefix is needed. 

These two use more than one level so the `TRACE#` prefixes are used: 
`afw.math.warp`
`afw.math.convolve`